docs/api-docs/getting-started/api-status-codes.md

```diff
@@ -1,7 +1,7 @@
 # API Status Codes
 
 
-The BigCommerce API responds to requests with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.
+The BigCommerce API responds to a request with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.

```

Comment:
```suggestion
The BigCommerce API responds to each request with an HTTP status code that depends on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.

```

Reply:
@bc-tgomez -- I like the shift to singular request.  How do you feel about this?